### ISSUE TITLE
Long delay before receiving login link

### ISSUE THREAD

**Issue by ann-kilzer** [Devographics/Monorepo#325]
I use a `proton.me` domain. Tried to log in twice, and the login link arrives the next morning and the token is expired

I understand that there is some unknown time in mail servers, but it seems messages are taking 12+ hours to arrive.

**Comment by SachaG** [comment]
Oh let me look into it. In the meantime you can also take the survey as a guest, without entering an email address.

**Comment by pwim** [comment]
Thanks for reporting the issue!

From looking at our mail provider's logs, I see that we sent a single mail to you, on the night of October 6th. Your mail server returned `smtp;250 2.0.0 Ok: queued`. 

That doesn't mean much though. Many email providers will accept emails that they're flagging as spam. 

If I had to guess, proton flagged our email as suspicious, and so put it into some sort of a quarantine, to see if we sent more emails like it to their users. When we didn't, it go released quarantine. This is pure speculation on my part though.

I have confirmed that emails can be sent to my own `proton.me` account, so hopefully it was just a temporary measure.

It doesn't explain to me why you tried to login twice, but only received a single email. Maybe @SachaG has an idea?

One thing I did notice is the email we're sending is very barebones. With only the content

> **TokyoDev Developer Survey**
> 
> [Log in to the survey.](#) 

Perhaps if we added a bit more text to the email, it'd be less likely to be flagged?

Then again, when reviewing notion's email for inspiration, it isn't as though they're having much more

> **Login**
>
> [Click here to log in with this magic link](#)
> Or, copy and paste this temporary login code:
> [Code]
> If you didn't try to login, you can safely ignore this email.
> Hint: You can set a permanent password in Settings & members → My account.
> 
> Notion.so, the all-in-one-workspace
> for your notes, tasks, wikis, and databases.

One more thing that @SachaG could investigate is what percentage of login links get clicked on, and if this is different with the TokyoDev survey vs the other devographics ones.
